rtl/eclrtl/rtlrecord.cpp

```diff
@@ -208,25 +182,39 @@ RtlRecord::RtlRecord(const RtlFieldInfo * const *_fields, bool expandFields) : f
         {
             const RtlFieldInfo * * allocated  = new const RtlFieldInfo * [numFields+1];
             fields = allocated;
-            const RtlFieldInfo * * target = expandNestedRows(allocated, originalFields);
+            const RtlFieldInfo * * target = expandNestedRows(nullptr, allocated, originalFields);
```

Comment:
This might cause problems/inefficiencies since this is used by generated code with > 3 variable width fields.  Needs more investigation/